DetergentDoseCalculator/DetergentDoseCalculator.csproj

```diff
@@ -1,15 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
   </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.TestPlatform" Version="17.2.0" />
-    <PackageReference Include="Microsoft.UnitTestFramework.Extensions" Version="2.0.0" />
-  </ItemGroup>
-
```

Comment:
A csproj file defines a C# project. You can think of a project as a reusable module that can be used by many other projects. You want to separate these out so you can reuse them, such as making an iPhone and Android app, they'll be different code bases, but use the same logic. It's also useful in your case because you have unit tests. When you deploy this out you don't want those unit testing libraries to be included, it takes more space and can introduce security vulnerabilities.

Reply:
The specific change here removes the testing libraries, because your calculator code doesn't use it and shouldn't use it.